src/app/cube/shared/footer/footer.component.html

```diff
@@ -22,11 +22,11 @@ <h3>Help</h3>
       <div>
         <h3>General Information</h3>
         <a aria-label= "Clickable About Us link" href="http://clark.center/about" >{{copy.ABOUT}}</a>
-        <a aria-label="Clickable Blog link" href="https://securedteam.notion.site/Featured-Blog-Posts-9ebff45f06f84a0a83c682f59472f106" target="_blank">{{copy.BLOG}}</a>
+        <a aria-label="Clickable Blog link" href="https://blogs.clark.center/about" target="_blank">{{copy.BLOG}}</a>
```

Comment:
Why us this pointing to about?

Reply:
No particular reason. Do you think it should be the home page?